system_test.sh

```diff
@@ -37,6 +46,18 @@ clear_tmp='rm -rf $tmp'
 trap "$clear_tty; $clear_tmux; $clear_tmp" EXIT
 trap 'printf "\nTerminated by SIGINT at line $LINENO\n"; exit 1' INT
 
+# required binary
+
+ENTR="$(./entr 2>&1)"
+if echo "$ENTR" | grep -qF release
+then
+	echo "Testing Entr " "$(echo "$ENTR" | grep -F release)"
+else
+	echo $ENTR
+	echo "Entr binary did not run as expected"
+	exit 1
+fi
+
```

Comment:
Can replace this with `make test`? The idea behind that make target is to "turn the crank" once just to make sure it's a working inary

Reply:
This section is needed to ensure specifically that `system-test.sh` is testing the correct binary and that is the recently built one specifically.